axelar-chains-config/info/stagenet.json

```diff
@@ -1765,6 +1765,79 @@
       "confirmations": 1,
       "finality": "finalized",
       "approxFinalityWaitTime": 1
+    },
+    "hedera": {
+      "name": "Hedera",
+      "timeout": 50000,
```

Comment:
why is this being overridden here? is it for rpc timeout (in ms?)? Also, move it a bit lower compared to other important fields